Realm/RLMObjectBase.mm

```diff
@@ -410,6 +412,21 @@ - (id)objectiveCMetadata {
     return nil;
 }
 
+#pragma mark - Key Path Strings
+
+- (void)prepareForRecording {
```

Comment:
This function probably could be written in Swift instead and if so should be as it's swift-specific.